File: cpp/day-05.cpp
#include <iostream>
using namespace std;
int main()
{
     int arr[] = {7, 4, 6, 2, 3, 35, 5, 8, 9, 1};
     int big = arr[0];
     int big_2 = 0;
     for (int x : arr)
     {
          if (x > big)
          {
               big_2 = big;
               big = x;
          }
     }
     cout << "=========================Find Biggest and Second Biggest Element==========================\n==========================================================================================\n";
     cout << "Given Array: " << endl;
     for (int x : arr)
     {
          cout << x << "\t";
     }
     cout << endl;
     cout << "The biggest element is: " << big << endl;
     cout << "The second biggest element is: " << big_2 << endl;
}

File: python/day-05.py
arr=[700, 4, 6, 2, 3, 35, 5, 8, 9, 10,60,89,53,23,45]
# Find the maximum value in the array and segond maximum value in the array
max1=arr[0]
max2=float('-inf')
j=1
for i in arr:
     if i>max1:
          max2=max1
          max1=i
     elif i > max2 and i != max1:
        max2 = i

print(f"Maximum value in the array is :{max1}")
print(f"Second Maximum value in the array is :{max2}")
